tests/project/package.json

```diff
@@ -1,4 +1,9 @@
 {
+  "name": "@godot-js/tests",
```

Comment:
I think this ought to be a PNPM lockfile for consistency since we're using PNPM elsewhere.